syntax = "proto3";

package embedding;

// EmbeddingRequest is the payload to embedding creation
message EmbeddingRequest {
    repeated string inputs = 1;
}

// EmbeddingResponse are what's returned by the gRPC service
message EmbeddingResponse {
    repeated string embeddings = 1;
}

service GenerateService {
    rpc CreateEmbedding(EmbeddingRequest) returns (EmbeddingResponse);
}
